codeview/src/main/java/io/github/kbiakov/codeview/adapters/CodeWithDiffsAdapter.kt

```diff
@@ -11,16 +13,18 @@ import io.github.kbiakov.codeview.views.LineDiffView
  *
  * @author Kirill Biakov
  */
-class CodeWithDiffsAdapter : AbstractCodeAdapter<DiffModel> {
+open class CodeWithDiffsAdapter : AbstractCodeAdapter<DiffModel> {
```

Comment:
You are used Kotlin ```open class```, but I not found related open methods and it's final class which is not necessary to inherit. Can you please explain your choice?